samples: bluetooth: locator_tag: Move speaker handling to LED thread (#27887)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  Support for this target was already dropped during the IronSide SE migration in the |NCS| v3.1.0 release.

* :ref:`fast_pair_locator_tag` sample:

>   * Updated the UI thread handling on platforms with a speaker by moving the speaker control into the common indication thread.

Reviewer: let's use terminology from FP Locator tag Readme

```suggestion
  * Updated the UI thread handling for reference board targets with a speaker by moving the speaker control into the common indication thread.
```

What about the fix changelog?

```
  * Fixed the ringing status indications with the green LED flashes for reference board targets.
    A ringing status indication was often skipped during the motion detection event.
```

Reviewer: mention the fix in the commit description?

  Support for this target was already dropped during the IronSide SE migration in the |NCS| v3.1.0 release.

* :ref:`fast_pair_locator_tag` sample:

  * Updated the UI thread handling for reference board targets with a speaker by moving the speaker control into the common indication thread.
    The signaling is now done using the :ref:`zephyr:events` API.

>   * Fixed the ringing status indications with the green LED flashes for reference board targets.

Reviewer: blank line before fixed
